lesson15/main.js

```diff
@@ -86,41 +88,49 @@ const displayModal= () => {
   overlay.style.display= "block";
 };
 
+const resetInput = () => {
+  const inputFields = document.getElementsByClassName("js-input");
+
+  for (errorMessage of errorMessages) {
+    errorMessage.textContent =""
+  };
+  for(inputField of inputFields) {
+    inputField.value = ""
+    inputField.style = "border border-slate-600 "
+  };
```

Comment:
IMO
I thought it was necessary to declare a variable in the for of.
Please check the error text in codesandbox every time as it is helpful.

```suggestion
  for (const errorMessage of errorMessages) {
    errorMessage.textContent =""
  };
  for(const inputField of inputFields) {
    inputField.value = ""
    inputField.style = "border border-slate-600 "
  };
```
![スクリーンショット 2023-01-08 18 48 03](https://user-images.githubusercontent.com/51282804/211189886-fa38791d-65f8-4cc4-a770-c3cb9db9dc9f.png)

Reply:
Thank you for the feedback, 
I was not aware that for...of method need "const" to define an element. 
fixed that issue.
Kindly check the commit 
6ad544c96b5c4e21ce5ff7a81fa84174f72e7142